shared/pkg/backendclient/client.go

```diff
@@ -208,7 +209,8 @@ func (b *BackendClient) PostScanResult(ctx context.Context, scanResult models.Ta
 }
 
 func (b *BackendClient) PatchScan(ctx context.Context, scanID models.ScanID, scan *models.Scan) error {
-	resp, err := b.apiClient.PatchScansScanIDWithResponse(ctx, scanID, *scan)
+	params := models.PatchScansScanIDParams{}
```

Comment:
I was going to handle this as a separate task, there are a lot of places that can be updated to use IfMatch, but a lot of it will change with the reconciler refactors so I was going to hold off for now.

Reply:
just weird that we are adding a return status in the API but not handling it in the code, but if you know it should be done and we have an issue for that, i'm fine with it.